bin/yaml-format.py

```diff
@@ -30,9 +30,14 @@ def description() -> str:
 
     def format(self, input_str: str) -> str:
         """Opinionated format YAML file."""
-        obj = yaml.load(input_str)
         if self.args.add_test_metadata:
-            self._add_test_metadata(obj)
+            # Since formatting integration test templates cause failure,
+            # temporary workaround to add test metadata to integration test templates
+            if "SamTransformTest" in input_str:
+                return input_str
+            return input_str + "\nMetadata:\n  SamTransformTest: true\n\n"
```

Comment:
Are we doing it here or adding to each test file? Though writing yaml like this isn't convenient it seems more efficient then appending it to 100+ files

Reply:
Sorry meant to close this PR